src/plugins/integrations/AWSIntegration.ts

```diff
@@ -185,10 +188,11 @@ class AWSIntegration implements Integration {
               activeSpan.setTag(SpanTags.TOPOLOGY_VERTEX, true);
               activeSpan.setTag(SpanTags.TRIGGER_DOMAIN_NAME, LAMBDA_APPLICATION_DOMAIN_NAME);
               activeSpan.setTag(SpanTags.TRIGGER_CLASS_NAME, LAMBDA_APPLICATION_CLASS_NAME);
+              activeSpan.setTag(AwsDynamoTags.TABLE_NAME, tableName);
             }
           } else if (serviceName === 's3') {
             const operationType = S3RequestTypes[operationName];
-            const bucketName = operationType ? request.params.Bucket : AWS_SERVICE_REQUEST;
+            const bucketName = koalas(request.params.Bucket, AWS_SERVICE_REQUEST);
```

Comment:
If bucket name is not valid for the request, it is ok to use `AWS_SERVICE_REQUEST` as the default span name, but it should not be used for `AwsS3Tags. OBJECT_NAME` tag. So you may handle this by introducing another local variable for span name.

Reply:
Done. Introduced a spanName variable.